auth/credentials/oauth_test.go

```diff
@@ -218,10 +223,10 @@ func TestOAuthClient_FetchToken_FailureHtmlContentType(t *testing.T) {
 		BaseURL:      &mockServer.URL,
 	})
 
-	// Call GetAccessToken expecting an error
-	_, err := client.GetValidToken()
+	// Call GetValidToken expecting an error
+	_, err := client.Token()
 	assert.Error(t, err)
-	assert.Contains(t, err.Error(), "Failed to obtain Access Token when fetching new OAuth Token from remote server")
+	assert.Contains(t, err.Error(), "oauth2: cannot fetch token: 403 Forbidden")
 }
 
 // Test CustomTransport to ensure Token is injected into requests
```

Comment:
not changed in this PR, but prefer t.Cleanup to defer in tests, e.g.: https://brandur.org/fragments/go-prefer-t-cleanup-with-parallel-subtests